roles/web/tasks/setup_virtualenv.yml

```diff
@@ -8,8 +8,11 @@
   command: virtualenv -p {{ virtualenv_python_version }} {{ virtualenv_path }}
            creates={{ virtualenv_path }}/bin/activate
 
-- name: Ensure gunicorn is installed
-  pip: virtualenv={{ virtualenv_path }} name=gunicorn
+- name: Ensure gunicorn and pycurl are installed in the virtualenv
+  pip: virtualenv={{ virtualenv_path }} name={{ item }}
+  with_items:
+    - gunicorn
+    - pycurl
```

Comment:
I'm actually not sure why I have `pycurl` and if it was a requirement to install a package or required for a task/Ansible module, any reason we moved the installation here?